main.js

```diff
@@ -57,6 +57,12 @@ function createWindow () {
   globalShortcut.register('CommandOrControl+Shift+2', () => {
     mainWindow.webContents.send('play', 'ping')
   })
+  globalShortcut.register('Alt+Shift+F', () => {
+    fullscreenToggle(mainWindow, false)
+  })
+  globalShortcut.register('esc', () => {
```

Comment:
There is an issue here. It is registered as the global event listener so even if Esc is pressed for some other browser, it will be called here as well.

Reply:
Good catch! Looking in the docs, the only other way I see to create a shortcut is by means of a menu item. Would simply checking if the window is in focus before calling the event handler suffice or is there a smarter way to make the shortcut?

Reply:
There are total 3 ways to detect keyboard shortcuts, first is globalShortcut, LocalShortCut through Menu and keyboard bindings on renderer process. The last one is not possible in our case because we have two types of media supports, first where we embed the content and second where we directly open a link. The cases of the latter one are very few, for instance, if we have to add support for providers like Netflix, we can embed and have to open the link directly. Therefore, I think the best option would be MenuItem as we already have a TrayIcon, you can simply add one more property with "Exit Full Screen" and keep it static for now, we will make it change between "Enter/Exit" in next PRs because I think we have to start modularizing our main.js now